app/models/manageiq/providers/ansible_tower/shared/automation_manager/job.rb

```diff
@@ -134,9 +134,10 @@ def raw_status
     raise MiqException::MiqOrchestrationStatusError, err.to_s, err.backtrace
   end
 
-  def raw_stdout
+  def raw_stdout(format = nil)
```

Comment:
Do we need a comment to prevent this from being cleaned up?  Or can we change the caller to not pass a nil?